license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -245,6 +280,10 @@
       "bundleName": "bsd3clause",
       "licenseNamePattern": "^The 3-Clause BSD License$"
     },
+    {
+      "bundleName": "bsd3clause",
+      "licenseNamePattern": "^BSD License$"
```

Comment:
issue: This is a different BSD license. Probably this one: https://spdx.org/licenses/BSD-1-Clause.html. Can we check the license text?

Reply:
Same here 
"flatdict","4.0.1","BSD License" - in pip-licenses report
License: BSD License (BSD 3-Clause License) - pypi.org

https://github.com/gmr/flatdict/blob/master/LICENSE - license itself